app/mm-redux/types/integrations.ts

```diff
@@ -106,13 +110,9 @@ export type DialogElement = {
     min_length: number;
     max_length: number;
     data_source: string;
-    options: Array<{
-        text: string;
-        value: any;
-    }>;
+    options: Array<DialogOption>;
 };
 export type InteractiveDialogConfig = {
-    app_id: string;
```

Comment:
This change seems unrelated.

Reply:
I'm not sure when this field should be removed. It is an artifact of an early implementation of the apps modal